zipline/utils/api_utils.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2015 Quantopian, Inc.
```

Comment:
Regarding module names, when should I put a new utility in `utils/api_support.py` versus `utils/api_utils.py`?  Do api_utils work with non-`api_method`s?